src/hub/Hub.sol

```diff
@@ -347,129 +337,105 @@ contract Hub is Multicall, Auth, Recoverable, IHub, IHubGatewayHandler {
         accounts[4] = newAccountId(prefix, uint8(AccountType.Expense));
         accounts[5] = newAccountId(prefix, uint8(AccountType.Liability));
 
-        createAccount(accounts[0], true);
-        createAccount(accounts[1], false);
-        createAccount(accounts[2], false);
-        createAccount(accounts[3], false);
-        createAccount(accounts[4], true);
-        createAccount(accounts[5], false);
-
-        holdings.create(unlockedPoolId, scId, assetId, valuation, isLiability, accounts);
-    }
-
-    /// @inheritdoc IHub
-    function updateHolding(ShareClassId scId, AssetId assetId) public payable {
```

Comment:
We can not remove that one. We need to be able to update the value of a holding!